src/games/strategy/engine/framework/map/download/DownloadRunnable.java

```diff
@@ -76,6 +76,7 @@ private void downloadFile() {
         }
       } catch (final Exception e) {
         error = e.getMessage();
+        ClientLogger.logError("Error while Parsing:", e);
```

Comment:
It's good practice to keep bug fixes minimized and on their own commit.  Plays well with commit history/rollbacks, etc.. This one line IMO ideally would have been the only item in this PR. Note that above we are talking about assumptions of the structure, which in terms of PRs is unrelated to how we handle the case when the YAML is not valid